tests/conftest.py

```diff
@@ -0,0 +1,30 @@
+from functools import wraps
+
+import astroid.builder
+import astroid.nodes
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--test-roundtrip-persistence", type=bool, default=True,
```

Comment:
Eventually, I think we'll add a tox section to re-run all tests with this enabled. For now it's just enabled so I can test easily.